x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataType.java

```diff
@@ -126,9 +127,9 @@
 
 
     /**
-     * Elasticsearch type name
+     * Lowercase type name
      */
-    public final String esType;
+    public final String esSQLType;
```

Comment:
I may be wrong, but why not `esSqlType` name?

Reply:
sure